cypress/integration/common/steps.js

```diff
@@ -1,6 +1,6 @@
 import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps'
 import find from 'lodash/find'
-import { eq } from 'semver';
+import { eq } from 'semver'
```

Comment:
Do we use that Import at all?

Reply:
I'm not sure, as this is not code I wrote or am yet familiar with...

Reply:
I removed it for now, we should see a failing cypress test if anything breaks